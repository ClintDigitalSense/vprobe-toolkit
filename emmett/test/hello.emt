VMM1Hz logint(1);
