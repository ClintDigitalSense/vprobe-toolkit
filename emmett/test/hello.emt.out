(version 0.2)

 ;;;;;
(vprobe VMM1Hz ( logint    1 ))